### ISSUE TITLE
Why this library?

### ISSUE THREAD

**Issue by izelnakri** [starbeamjs/esyes#1]
I see that currently you are suppressing the node warning is this the reason of this library? It wasn't clear from the project description . I suggest using `ts-node` instead unless I'm missing something :)

P. S I'm just happily getting out of the nix/nixOS/home-manager k8s cluster rabbit hole I got myself into the last few months. I do see the light in the end of the tunnel in few weeks/months & will continue with `qunitx-cli` test enhancements, finish `memoria` beta release  and start working on `emberx`

**Comment by wycats** [comment]
The TL;DR is: it uses the same setup as `tsx` under the hood. When written, it was the first version of "run my TS without deprecation notices from Node," but other tools have caught up.

It also implements `import.meta.env`, which is very valuable for setups that use Vite for browser, test and build, but still have `.ts` files that you want to just run from Node.

My personal goal is to have a fairly simple "strip the types out and add in the bare minimum transformations so that my Vite-authored code works directly from Node." At some point, it may not be necessary, but I've found that this tool gives me what I need and tends to work for longer stretches of time and with more bleeding-edge TypeScript and Node features than the alternatives.

I'm not very attached to it and would be very happy to retire when it makes sense.
